src/Grid/CalendarView.vala

```diff
@@ -233,7 +234,9 @@ public class Maya.View.CalendarView : Gtk.Grid {
 
     /* Render new event on the grid */
     void add_event (E.Source source, ECal.Component event) {
-        event.set_data ("source", source);
+        /* The "source" data is added to events byt the CalendarModel. The grid must only show events that have
```

Comment:
> `/* The "source" data is added to events byt the CalendarModel.`

It's just a comment, but there's a small typo in here: "byt the"

Reply:
Fixed -thanks.